src/main/java/com/minecolonies/coremod/client/gui/DialogDoneCancel.java

```diff
@@ -0,0 +1,192 @@
+package com.minecolonies.coremod.client.gui;
+
+import com.minecolonies.blockout.controls.Button;
+import com.minecolonies.blockout.controls.Label;
+import com.minecolonies.blockout.controls.Text;
+import com.minecolonies.blockout.Loader;
+import com.minecolonies.blockout.views.Window;
+import com.minecolonies.coremod.lib.Constants;
+import com.minecolonies.blockout.OverlayView;
+import com.minecolonies.coremod.util.Log;
+import java.util.function.ObjIntConsumer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.function.ObjIntConsumer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)